client/src/pages/History.js

```diff
@@ -55,7 +55,7 @@ function History() {
 	}, [ctx.token, setLoadedData]);
 
 	return (
-		<Layout title="History">
+		<Layout title="Scheduled">
```

Comment:
Changed this to `Ongoing` so that we can put both both recurring and scheduled in a single page